projects/ngx-drag-to-select/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "ngx-drag-to-select",
-  "version": "2.1.0",
+  "version": "2.2.0",
   "description": "A lightweight, fast, configurable and reactive drag-to-select component for Angular 6 and beyond",
   "sideEffects": false,
   "peerDependencies": {
-    "@angular/core": ">=6.0.0",
-    "@angular/common": ">=6.0.0",
+    "@angular/core": ">=6.0.0 || >=7.0.0",
```

Comment:
I think we can leave this for now. Would you mind checking if this lib still works in an Angular 6 project once the lib internally uses Angular 7? It should work seemingless because there were no breaking changes and v7 should be downwards compatible.